Source/WebKit/UIProcess/API/wpe/WPEView.cpp

```diff
@@ -152,6 +152,7 @@ View::View(struct wpe_view_backend* backend, const API::PageConfiguration& baseC
 View::~View()
 {
     m_compositingManagerProxy.finalize();
+    wpe_view_backend_destroy(m_backend);
```

Comment:
The backend ownership is not intended for this class. Sure this fixes the leak, but the ownership should be managed alongside the WKView ownership.

Reply:
i.e. this change would not be necessary, instead the wpe_view_backend lifetime should be managed inside the plugin.

Reply:
@zdobersek  this change makes wpe-2.22 behave similarly to wpe-20170728. So, my understanding is that from wpe-2.22 the t WKViewCreate user takes ownership over the backend, is that correct?

Reply:
OK, I think this should be fine for the WK2 C API. Let me just check before merging.